### PR TITLE
Package updates

### PR DESCRIPTION
**Pull request by IanCStewart**
- Updated main packages
- Updated eject packages of docs/package.json

*This fixes the security error*

_**http://russelljanderson.com/updating-create-react-app/#tldr used this guide to update our ejected CRA app**_

### CODE REVIEW

File: src/emoji-filter/component.jsx [hunk 1]
```diff
@@ -148,7 +150,7 @@ class EmojiFilter extends Component {
     }
 
     if (key === 13) {
-      this.handleSelect(event, this.state.emoji[selectedIndex], selectedIndex);
+      this.handleSelect(event, this.state.emoji[selectedIndex], selectedIndex); // eslint-disable-line react/destructuring-assignment
     }
```
Comment (peterkuiper) [inline src/emoji-filter/component.jsx]: Why do you still use this.state.emoji here?

```
const { open, emoji } = this.state;
```

File: src/message-input/index.jsx [hunk 2]
```diff
@@ -236,15 +236,15 @@ class MessageInput extends Component {
     }
 
     return (
-      <section style={getStyles.root(disabled, height, this.state.multiLine, style)}>
+      <section style={getStyles.root(disabled, height, this.state.multiLine, style)}> {/* eslint-disable-line react/destructuring-assignment, max-len */}
         {
```
Comment (peterkuiper) [inline src/message-input/index.jsx]: Why don't you:

```
const { multiLine } = this.state;
```

File: src/themeable/index.jsx [hunk 3]
```diff
@@ -37,7 +37,7 @@ const themeable = options => ChildComponent => (
 
     render() {
       const propName = (options && options.propName) || 'color';
-      const customPropName = { [propName]: this.state.color };
+      const customPropName = { [propName]: this.state.color }; // eslint-disable-line react/destructuring-assignment
```
Comment (peterkuiper) [inline src/themeable/index.jsx]: Why don't you:

```
const { color } = this.state;
```

Comment (IanCStewart) [reply src/themeable/index.jsx]: Good find! I have adjusted this.

File: src/message-input/index.jsx (hunk 2 above)
Comment (IanCStewart) [reply src/message-input/index.jsx]: Multiline is also in `this.props`. I have chosen to add `eslint-disable-line react/destructuring-assignment` to the `this.state` version of multiline.

File: src/emoji-filter/component.jsx (hunk 1 above)
Comment (IanCStewart) [reply src/emoji-filter/component.jsx]: `emoji` is predefined on line 16 `import emoji from '../emoji-menu/emoji';`.

**Pull request merged**